docs/source/installation.rst

```diff
@@ -83,127 +83,81 @@ Note: not all soft dependencies of ``sktime`` are also available on ``conda-forg
 
-    every time the source code of a compiled extension is changed (for
-    instance when switching branches or pulling changes from upstream).
+To develop ``sktime`` locally, or to contribute to the project, you need to set up:
 
-Step 2 - Building binary packages and installers
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* a local clone of the ``sktime`` repository.
+* a virtual environment with an editable install of ``sktime`` and its developer dependencies.
 
-The ``.whl`` package and ``.exe`` installers can be built with:
-
-.. code-block:: bash
-
-    pip install build
-    python -m build --wheel
-
-The resulting packages are generated in the ``dist/`` folder.
-
-Contributor or 3rd party extension developer setup
---------------------------------------------------
+The following steps guide you through the process:
 
 1. Follow the Git workflow: Fork and clone the repository as described in [Git and GitHub workflow](https://www.sktime.net/en/stable/developer_guide/git_workflow.html)
 
 2. Set up a new virtual environment. Our instructions will go through the commands to set up a ``conda`` environment which is recommended for sktime development.
-This relies on an `anaconda installation <https://www.anaconda.com/products/individual#windows>`_. The process will be similar for ``venv`` or other virtual environment managers.
+The process will be similar for ``venv`` or other virtual environment managers.
+
+  .. warning::
+       Using ``conda`` via one of the commercial distributions such as Anaconda
+       is in general not free for commercial use and may incur significant costs or liabilities.
+       Consider using free distributions and channels for package management,
+       and be aware of applicable terms and conditions.
 
-In the ``anaconda prompt`` terminal:
+In the ``conda`` terminal:
 
 3. Navigate to your local sktime folder, :code:`cd sktime` or similar
 
-4. Create a new environment with a supported python version: :code:`conda create -n sktime-dev python=3.8` (or :code:`python=3.11` etc)
+4. Create a new environment with a supported python version: :code:`conda create -n sktime-dev python=3.11` (or :code:`python=3.12` etc)
 
    .. warning::
-       If you already have an environment called "sktime-dev" from a previous attempt you will first need to remove this.
+       If you already have an environment called ``sktime-dev`` from a previous attempt you will first need to remove this.
 
 5. Activate the environment: :code:`conda activate sktime-dev`
 
 6. Build an editable version of sktime.
-In order to install only the dev dependencies, :code:`pip install -e .[dev]`
+In order to install only the dev dependencies, :code:`pip install -e ".[dev]"`
 If you also want to install soft dependencies, install them individually, after the above,
-or instead use: :code:`pip install -e .[all_extras,dev]` to install all of them.
-
-    .. note::
+or instead use: :code:`pip install -e ".[all_extras,dev]"` to install all of them.
```

Comment:
Will it make sense to suggest `all_extras_pandas2` instead of `all_extras`?

Reply:
hm, we have not advertised that dep set so far, though.

Reply:
Given it's more than 1.5 years from pandas v2 release, I think we can consider to start focusing only on v2. That may include rename all_extras_pandas2 to all_extras and removing current all_extras, and/or drop v1 specific testing in CI.